slack.md

```diff
@@ -5,31 +5,10 @@ title: Chat Local
 header-img: 'img/slack-action.jpg'
 ---
 
-Come hang out with local Syracuse developers in the OpenHack Syracuse Slack
-group.
+Come hang out with local Syracuse developers in the OpenHack Syracuse Slack.
```

Comment:
Might want to specifiy channel here. Reads a little weird. i.e.

`Come hang out with local Syracuse developers in the OpenHack Syracuse Slack channel.`